egs/wsj/s5/utils/data/get_utt2dur.sh

```diff
@@ -68,7 +70,10 @@ else
       echo  "$0: wav-to-duration is not on your path"
       exit 1;
     fi
```

Comment:
I think at this point, if read_entire_file is false, you should check whether the wav.scp has any of the offending sox commands that tend to create this problem, and if so, set it to true and print a message saying why.